include/util.h

```diff
@@ -70,15 +105,26 @@ int ldlt(Eigen::MatrixXd* A, Eigen::MatrixXd* L, Eigen::VectorXd* d, Eigen::Vect
  * U is not
  * Return != 0 if getf failed (singular)
  */
-int getf(Eigen::MatrixXd* A, Eigen::VectorXi* swap);
+int64_t getf(Eigen::MatrixXd* A, Eigen::VectorXi64* swap);
 
 /**
  * A = P L U Q
  * A <- [L\U] (lower and upper)
  * L is unit diagonal
  * U is not
  */
-void fpgetf(Eigen::MatrixXd* A, Eigen::VectorXi* p, Eigen::VectorXi* q);
+template <typename VEC1, typename VEC2>
+void fpgetf(Eigen::MatrixXd* A, VEC1* p, VEC2* q) {
```

Comment:
All of those templates are because we have `VEC{1,2}` of different types? Can't we just always take `VectorXi64`? (that would reduce how much changes there are)

Reply:
I vaguely remember there being obstacles to taking only VectorXi64. But I'll try again, and report back.